patches/react-native-image-picker+5.1.0.patch

```diff
@@ -0,0 +1,134 @@
+diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java
+index 89b69a8..9345508 100644
+--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java
++++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java
+@@ -29,6 +29,121 @@ public class ImagePickerModuleImpl implements ActivityEventListener {
+     public static final int REQUEST_LAUNCH_VIDEO_CAPTURE = 13002;
+     public static final int REQUEST_LAUNCH_LIBRARY = 13003;
+
++    // Avoid image picker to show svg images on Android, to get the behaviour to be same as on iOS.
++    // Since, we can't exclude an image format, we instead allowlist all the possible image formats,
++    // excluding the 'image/svg+xml' mime type.
```

Comment:
Just some comments clarification:

Prevent svg images from being selected as they are not supported (Image component does not support them) and also because iOS does not allow them to be selected (for consistency).

Since, we can't exclude a mime type, we instead allow all image mime types except 'image/svg+xml'. Image mime types are generated by merging the Android image mime type support and the IANA media-types lists.